samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -147,8 +147,8 @@ static void recv_cb(struct bt_le_per_adv_sync *sync,
 		    const struct bt_le_per_adv_sync_recv_info *info,
 		    struct net_buf_simple *buf)
 {
+	static char data_str[BT_GAP_ADV_MAX_EXT_ADV_DATA_LEN];
```

Comment:
The hexadecimal representation is two chars per byte plus null-terminator. I think this should be:
```suggestion
	static char data_str[BT_GAP_ADV_MAX_EXT_ADV_DATA_LEN * 2 + 1];
```

Reply:
I do not understand, could you rephrase your question?